src/main/webapp/app/exercises/file-upload/participate/file-upload-submission.component.ts

```diff
@@ -186,6 +187,15 @@ export class FileUploadSubmissionComponent implements OnInit, ComponentCanDeacti
         }
     }
 
+    /**
+     * Find "Unreferenced Feedback" item for Result, if it exists.
+     */
+    get unreferencedFeedback(): Feedback[] | undefined {
+        if (this.result && this.result.feedbacks) {
+            return this.result.feedbacks.filter((feedbackElement) => feedbackElement.reference == undefined && feedbackElement.type === FeedbackType.MANUAL_UNREFERENCED);
```

Comment:
here you could use `!feedbackElement.reference`

Reply:
Probably really just a matter of taste but I would prefer to see the == undefined here :P

Reply:
👍  c01428c